src/main/resources/web_de.properties

```diff
@@ -43,7 +45,7 @@ help = Hilfe
 table.html.empty = Keine Inhalte gefunden
 template.html.additionalActions = Weitere Aktionen
 template.html.confirmHeader = Sind Sie sicher?
-template.html.confirmText = Drücken Sie 'Ja', um fortzufahren oder 'Abbrechen', um zurückzugehen.
+template.html.confirmText = Drücken Sie 'Ja', um fortzufahren oder Abbrechen, um zurückzugehen.
```

Comment:
…

Reply:
? Wie ist das passiert ?????
Ich habe ganz gezielt nur die 2 Copy-Feature Texte angepasst????

Reply:
Wtf in Lokalise steht es so drin:
![Bildschirmfoto 2023-09-22 um 15 21 03](https://github.com/scireum/sirius-web/assets/106659278/a01d66ca-569a-474b-b0e7-d797d1d867a7)

Das "Tool" macht mich fertig